### PR TITLE
[DOR-52][AO] add create case confirmation page

### PR DESCRIPTION
**Pull request by arturopala**

### CODE REVIEW

File: app/uk/gov/hmrc/traderservices/journeys/TraderServicesFrontendJourneyModel.scala [hunk 1]
```diff
@@ -48,10 +48,11 @@ object TraderServicesFrontendJourneyModel extends JourneyModel {
     def isVesselDetailsAnswerMandatory(importQuestions: ImportQuestions): Boolean =
       importQuestions.requestType.contains(ImportRequestType.Hold)
 
-    val maxFileUploadsNumber = 3
+    val maxFileUploadsNumber: Int = 10
```
Comment (Tasmine10ten) [inline app/uk/gov/hmrc/traderservices/journeys/TraderServicesFrontendJourneyModel.scala:51]: The actual change here is fine although I did notice that if you click continue after adding the max amount of files we get to a file not found/work in progress page. I messed about with this number locally to test and it seems to repeat this behaviour each time but only when it's the max number (the file types etc are allowed)

File: conf/application.conf [hunk 2]
```diff
@@ -172,7 +172,7 @@ urls {
 
 file-formats {
   max-file-size-mb = 10
-  approved-file-extensions = ".jpeg,.jpg,.png,.pdf,.doc,.docx,.xls,.xlsx,.odt,.odf,.tiff,.tif"
+  approved-file-extensions = ".jpeg,.jpg,.png,.pdf,.doc,.docx,.xls,.xlsx,.odt,.ods,.tiff,.tif"
```
[the remark this answers is not in the document]
Comment (arturopala) [reply conf/application.conf:175]: nope, the list of mime-types is up-to-date

File: conf/messages.en [hunk 3]
```diff
@@ -447,6 +447,17 @@ error.file-upload.unknown=The selected file could not be uploaded
 error.file-upload.try-again=The selected file could not be uploaded - try again
 error.uploadAnotherFile.required=Select yes if you want to upload another document
 
+# Confirmation messages
```
[the remark this answers is not in the document]
Comment (arturopala) [reply conf/messages.en:450]: no problem

File: app/uk/gov/hmrc/traderservices/journeys/TraderServicesFrontendJourneyModel.scala (hunk 1 above)
Comment (arturopala) [reply app/uk/gov/hmrc/traderservices/journeys/TraderServicesFrontendJourneyModel.scala:51]: good catch

File: conf/application.conf (hunk 2 above)
[the remark this answers is not in the document]
Comment (Tasmine10ten) [reply conf/application.conf:175]: Ah sorry yes you did mention!

File: app/uk/gov/hmrc/traderservices/journeys/TraderServicesFrontendJourneyModel.scala (hunk 1 above)
Comment (arturopala) [reply app/uk/gov/hmrc/traderservices/journeys/TraderServicesFrontendJourneyModel.scala:51]: fixed

File: conf/messages.en (hunk 3 above)
[the remark this answers is not in the document]
Comment (arturopala) [reply conf/messages.en:450]: fixed

**Review by Tasmine10ten — APPROVED** [review]

**Pull request merged**